pkg/backends/options/defaults.go

```diff
@@ -58,8 +58,8 @@ const (
 	DefaultMaxIdleConns = 20
 	// DefaultForwardedHeaders defines which class of 'Forwarded' headers are attached to upstream requests
 	DefaultForwardedHeaders = "standard"
-	// DefaullALBMechansimName defines the default ALB Mechanism Name
-	DefaullALBMechansimName = "rr" // round robin
+	// DefaultALBMechansimName defines the default ALB Mechanism Name
+	DefaultALBMechansimName = "rr" // round robin
```

Comment:
```suggestion
	DefaultALBMechanismName = "rr" // round robin
```

Reply:
Will need to change the references to this too, so i'll make a commit that includes `sim` -> `ism` here in the definition and also update the couple of places where it is referenced.